TradeItIosEmsApi/TradeItConnector.m

```diff
@@ -355,15 +358,21 @@ - (void)unlinkLogin:(TradeItLinkedLogin *)login
     [self.userDefaults setObject:accounts forKey:BROKER_LIST_KEYNAME];
 
     if (!localOnly) {
-        [self oAuthDeleteLink:login];
+        [self oAuthDeleteLink:login withCompletionBlock:completionBlock];
+    } else {
```

Comment:
I think it reads clearer to reverse these so `if (localOnly) {` instead of having the `!` case first